src/fairchem/core/trainers/base_trainer.py

```diff
@@ -112,8 +114,7 @@ def __init__(
         self.config = {
             "task": task,
             "trainer": name,
-            "model": aii(model.pop("name"), str),
-            "model_attributes": model,
```

Comment:
does this break backwards compatibility with old checkpoints? It looks like you updated load_checkpoint but I wanted to check?